packages/contentstack-dev-dependencies/src/spy.ts

```diff
@@ -1,5 +1,5 @@
 import sinon from 'sinon';
-import { test } from '@oclif/test';
+// import { test } from '@oclif/test';
```

Comment:
@cs-raj ,  remove unnecessary commented code

Reply:
Hi @aman19K this will be removed once all the test cases are finalised as I might be trying to add spy on fancy for the other plugins